src/Directory.Build.props

```diff
@@ -39,6 +39,8 @@
 	
 	<UNO_UWP_BUILD>true</UNO_UWP_BUILD>
 	<DefineConstants Condition="'$(UNO_UWP_BUILD)'!='true'">$(DefineConstants);HAS_UNO_WINUI</DefineConstants>
+    <DefineConstants Condition="'$(UnoTargetFrameworkOverride)'=='MonoAndroid11.0'">$(DefineConstants);CURRENT_PLATFORM_ANDROID</DefineConstants>
```

Comment:
For future-proofing I would also suggest adding `MonoAndroid12.0`, `net6.0-android` and `net6.0-ios` 👍